Dockerfile

```diff
@@ -1,11 +1,11 @@
 FROM openjdk:8
-COPY ./target/stuff-doer-1.0-SNAPSHOT-shaded.jar /usr/local/stuff-doer/stuff-doer-1.0-SNAPSHOT-shaded.jar
+COPY ./target/scala-2.11/stuff-doer.jar /usr/local/stuff-doer/stuff-doer.jar
 COPY ./src/main/resources/application.conf /usr/local/stuff-doer/application.conf
 COPY ./src/main/resources/logback.xml /usr/local/stuff-doer/logback.xml
 COPY ./src/main/resources/html/* /usr/local/stuff-doer/src/main/resources/html/
 COPY ./src/main/resources/mp3/* /usr/local/stuff-doer/src/main/resources/mp3/
 WORKDIR /usr/local/stuff-doer
 EXPOSE 9080
-CMD ["java", "-Dconfig.file=/root/stuff-doer/config/application.conf", \
+CMD ["java", "-Dconfig.file=/usr/local/stuff-doer/application.conf", \
```

Comment:
Please revert this line back to `/root/` otherwise it will break the production deployment. I'm working on fixing it in issue #113

Reply:
Opss, sorry about that